.chloggen/sqlserverreceiver-unexport.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/sqlserverreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Unexport sqlserverreceiver `MockPerfCounterWatcher`
```

Comment:
I'm sorry, it looks like I was wrong. I just checked if objects exported in `_test.go` files are actually available outside, appeared they are not. So this changelog isn't needed

Reply:
No problem, removed.